fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -344,6 +344,8 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	bool DD_ENABLE_REBALANCE_STORAGE_QUEUE_WITH_LIGHT_WRITE_SHARD; // Enable to allow storage queue rebalancer to move
 	                                                               // light-traffic shards out of the overloading server
 	double DD_WAIT_TSS_DATA_MOVE_DELAY;
+	bool DD_VALIDATE_SERVER_TEAM_COUNT_AFTER_BUILD_TEAM; // Enable to validate server team count per server after build
```

Comment:
Maybe add `SIM_` prefix?

Reply:
I feel we can turn on this knob even in a real cluster. Seems that adding SIM may be a bit confusing